internal/services/compute/linux_virtual_machine_scale_set_resource_scaling_test.go

```diff
@@ -250,6 +250,35 @@ func TestAccLinuxVirtualMachineScaleSet_scalingZonesMultiple(t *testing.T) {
 	})
 }
 
+func TestAccLinuxVirtualMachineScaleSet_scalingZonesUpdate(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_linux_virtual_machine_scale_set", "test")
+	r := LinuxVirtualMachineScaleSetResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.scalingZonesSingle(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("admin_password"),
+		{
+			Config: r.scalingZonesMultiple(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("admin_password"),
+		{
+			Config: r.scalingZonesSingle(data),
```

Comment:
My understanding is that we can add new zones but can't remove existing zones, so will this step recreate the resource? If so, we shouldn't have this step

Reply:
You're correct. It will recreate the resource. I have removed this step. Thanks.